src/components/stepper/index.js

```diff
@@ -4,7 +4,7 @@ import Button from "../button";
 import "./style.css";
 import { useState } from "react";
 
-const Stepper = ({ header, children, onComplete }) => {
+const Stepper = ({ header, children, onComplete, missingFields }) => {
```

Comment:
This is a step in the right direction but what may be even better is a simple `canProgress` boolean prop.

```js
const canProgress = profile.firstName.length !== 0 && profile.lastName.length !== 0

<Stepper header={<WelcomeHeader />} onComplete={onComplete} canProgress={canProgress}>
```